app/controllers/api/users.php

```diff
@@ -47,6 +47,14 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
         $email = \strtolower($email);
     }
 
+    // Makes sure this email is not already used in another identity
+    $identityWithMatchingEmail = $dbForProject->findOne('identities', [
+        Query::equal('providerEmail', [$email]),
```

Comment:
Are we making sure to attach only verified emails? Otherwise anyone could fake an email in a 3rd party provider and gain access to it on Appwrite.

Reply:
We're actually not really checking verified at all today because not all providers verify